### DISCUSSION TITLE
--citeproc issue with LaTeX and Docker

### DISCUSSION THREAD

**Discussion by thomjur** [jgm/pandoc#9874] [Q&A] [answered]
Hi everyone,

I am using the docker image pandoc/latex:3.1-ubuntu, and I hadn't touched it in a while. Recently, I had to recreate the image and all of a sudden my citeproc citations do not work anymore (I am using `--citeproc`). I couldn't find any changes in the pandoc release notes, and I am unsure when this has happened. Everything works fine with a previously created image (like a couple of months ago), but the former `authors like Munro \& Strohminger (\protect\hyperlink{ref-munrostrohminger2021}{2021})` now looks like this `authors like Munro \& Strohminger (\citeproc{ref-munrostrohminger2021}{2021})` in the intermediate TeX format. The references section has also changed. Can someone please help me with what to do? I am a LaTeX noob and most likely I just have to install different LaTeX packages, which are not yet included in the image?

Thank you for your help!

**Comment by jgm** [comment, marked as answer]
See commit 353177f9e7eadd60d66846325abd537b85fd46bb and #9031.
(pandoc 3.1.7)
Are you using a custom template? If so, you'll need to adjust it to incorporate changes that define the `\citeproc` macro.

**Reply by thomjur** [reply]
Great, thanks a lot, copying the respective part from the new template into our custom template solved the issue!

**Comment by tarleb** [comment]
Just adding that we maintain a `3.1.1` image, specifically because of the largish number of changes from 3.1.1 to 3.1.13 – the underlying OS shipped in those images changed, too. We usually only maintain one image per major version, but in this case the extra effort seemed warranted.

I recommend to pin the image to a minor version, so something like `pandoc/latex:3.1.1`, to avoid issues like this.
